common/static/common/js/discussion/utils.js

```diff
@@ -216,9 +216,21 @@
                     );
                 };
             }
+
+            if ($elem) {
+                $elem.prop("disabled", true);
+            }
+            if (params.$loading) {
+                if (params.loadingCallback) {
+                    params.loadingCallback.apply(params.$loading);
+                } else {
+                    self.showLoadingIndicator($(params.$loading), params.takeFocus);
```

Comment:
Curious why `$(params.$loading)` is needed - is `params.$loading` not already a jquery object?